addons/website_sale_digital/controllers/main.py

```diff
@@ -26,12 +26,14 @@ def payment_confirmation(self, **post):
     @http.route([
         '/shop/orders',
         '/shop/orders/page/<int:page>',
+        '/shop/orders/<model("sale.order"):order>',
     ], type='http', auth='user', website=True)
-    def orders_followup(self, page=1, **post):
+    def orders_followup(self, order=False, page=1, **post):
```

Comment:
Nope its not a bug. The argument was added so that we don't need another controller to display particular order having digital product and we can reuse the same controller and already defined template rendered by this controller.